### DISCUSSION TITLE
国内OSPO有多少？

### DISCUSSION THREAD

**Discussion by xgdyp** [X-lab2017/open-wonderland#188] [6️⃣ 提问题]
今天看到一个OSPO的数据，
![image](https://user-images.githubusercontent.com/37795442/189903313-1d28bca3-05d3-4c0d-8bf1-5a635ad5358a.png)
目前国内是只有这些OSPO嘛？

**Comment by will-ww** [comment]
对，主要的就这些，OSPO 都是大厂玩的，不过接下来会有不少也会成立 OSPO 的，包括大学、政府，也有机会~
